iml-wire-types/wire-types.rs

```diff
@@ -769,7 +771,7 @@ pub struct Filesystem {
     pub immutable_state: bool,
     pub label: String,
     pub mdts: Vec<Mdt>,
-    pub mgt: String,
+    pub mgt: Mgt,
```

Comment:
I think this should stay as a `String`, with the FS CLI needing a second fetch to get the Mgt.

Otherwise, we'll need to make `Filesystem` take a generic param (or duplicate the struct), as it's possible for the mgt field to be a `String`. 

For example, the following query code will represent the mgt field as a resource URI:

https://github.com/whamcloud/integrated-manager-for-lustre/blob/95e3b1992df5e12c1bb3988fdbd050bc3fd8930d/iml-warp-drive/src/cache.rs#L108-L112